drools-wb-screens/drools-wb-dtable-xls-editor/drools-wb-dtable-xls-editor-backend/src/main/java/org/drools/workbench/screens/dtablexls/backend/server/indexing/DecisionTableXLSXFileIndexer.java

```diff
@@ -19,9 +19,11 @@
 
 import javax.enterprise.context.ApplicationScoped;
 import javax.inject.Inject;
-
```

Comment:
Similar comment, please do not remove due to our code style: https://github.com/kiegroup/droolsjbpm-build-bootstrap/tree/master/ide-configuration/intellij-configuration/code-style